aerospike/datadog_checks/aerospike/aerospike.py

```diff
@@ -177,6 +203,60 @@ def get_info(self, command, separator=';'):
 
         return data.split(separator)
 
+    def collect_latency(self, namespaces):
+        data = self.get_info('latency:')
+
+        ns = None
+
+        metric_names = []
+
+        metric_values = []
+
+        while data:
+            line = data.pop(0)
+
+            while line.startswith("error-"):
```

Comment:
This loops does not seem handle the case when the last lines of `data` are made out of `error-` line.

Indeed, if that's the case we do break on line 220 but the `line` variable contains a string starting with `error-` and the code continue its execution

Reply:
Instead of this extra loop can we instead do:
```python
if line.startswith('error-'):
  continue
```
?